src/views/ApartmentApp/index.js

```diff
@@ -65,11 +76,11 @@ export default class ApartApp extends Component {
    */
   onApplicantRemove = profileToRemove => {
     if (profileToRemove && profileToRemove !== null) {
-      let applicantList = this.state.applicants; // make a separate copy of the array
-      let index = applicantList.indexOf(profileToRemove);
+      let applicants = this.state.applicants; // make a separate copy of the array
+      let index = applicants.indexOf(profileToRemove);
       if (index !== -1) {
-        applicantList.splice(index, 1);
-        this.setState({ applicants: applicantList });
+        applicants.splice(index, 1);
+        this.setState({ applicants });
```

Comment:
Should we have an error here in the case of index being -1 to warn that they are trying to remove someone who is not there?

Reply:
Yes, it is a check to ensure that `splice` does not get called with an invalid `index`, because otherwise it will "crash" the whole page. It was just the standard way that I saw this done when comparing multiple online guides, I guess technically you could use a `try ... catch` but that is somewhat less elegant.

Reply:
Okay, but should we have an error message for the user in the case of index being -1?

Reply:
Since the remove button only exists if the person is being displayed in the list, then there isn't really a need to put an error message (no need to worry about a user clicking a button that does not exist)

Reply:
That's fair.